### PR TITLE
feat: added 3D Carousel

### PR DESCRIPTION
**Pull request by suyogkokaje**
As the size of gif was too huge, I have added link of the video in the readme file. Please have a look.

### CODE REVIEW

File: Carousels/3D Carousel_01/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# 3D Carousel
+<h3> Tech Stack Used <img src = "https://media2.giphy.com/media/QssGEmpkyEOhBCb7e1/giphy.gif?cid=ecf05e47a0n3gi1bfqntqmob8g9aid1oyj2wr3ds3mg700bl&rid=giphy.gif" width = 32px> </h3>
+<p align="left"> <a href="https://www.w3.org/html/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg" alt="html5" width="40" height="40"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg" alt="css3" width="40" height="40"/> </a>   </p>
+<hr>
+
+<h2><a href="https://streamable.com/9sb97s">Preview</a></h2>
```
Comment (Ovenoboyo) [inline Carousels/3D Carousel_01/README.md:6]: Preview URL seems to be broken. Also if you want to show a gif in README you can simply use this syntax

<h2>"Alt text"</h2>

Sample gif

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

File: Carousels/3D Carousel_01/README.md (hunk 1 above)
Comment (suyogkokaje) [reply Carousels/3D Carousel_01/README.md:6]: URL is working for me 😅. After clicking on preview you will be redirected to https://streamable.com/9sb97s this url. As gif size was too large it was not loading in readme so I added link to that video 😅

Comment (suyogkokaje) [reply Carousels/3D Carousel_01/README.md:6]: @Ovenoboyo what should I do now?

Comment (Abirpal202049) [reply Carousels/3D Carousel_01/README.md:6]: Add the gif to your folder then Link it

Comment (suyogkokaje) [reply Carousels/3D Carousel_01/README.md:6]: Okay

Comment (Ovenoboyo) [reply Carousels/3D Carousel_01/README.md:6]: > URL is working for me 😅. After clicking on preview you will be redirected to https://streamable.com/9sb97s this url. As gif size was too large it was not loading in readme so I added link to that video 😅

My bad, streamable was being blocked by my ISP

Comment (suyogkokaje) [reply Carousels/3D Carousel_01/README.md:6]: @Abirpal202049   

Screenshot-1498

I got this. I also tried compressing gif with veed.io but its not allowing me to upload large size gif 😓

Comment (saikatmu31) [reply Carousels/3D Carousel_01/README.md:6]: @suyogkokaje if Large Size is not Supported Use some Online GIF Shrinker to Reduce the Size and then try attaching it.

Comment (suyogkokaje) [reply Carousels/3D Carousel_01/README.md:6]: @saikatmu31 I tried veed.io but its not allowing me to upload this karte file for compression 😓

Comment (suyogkokaje) [reply Carousels/3D Carousel_01/README.md:6]: large* file

**Comment by Ovenoboyo** [comment]
@suyogkokaje i have uploaded a preview gif. Please check it out and ill approve this PR

**Review by Ovenoboyo — APPROVED** [review]

**Comment by suyogkokaje** [comment]
> @suyogkokaje i have uploaded a preview gif. Please check it out and ill approve this PR

Thank you so much for helping @Ovenoboyo 😅. Looking good 😄.

**Review by Abirpal202049 — APPROVED** [review]

**Review by saikatmu31 — APPROVED** [review]

**Review by bhavesh-chaudhari — APPROVED** [review]

**Pull request merged**